pyproject.toml

```diff
@@ -88,8 +88,7 @@ pylinac = { version = "2.3.2", optional = true }  # groups = ["user", "dev", "do
 
 # Scikit-learn has an upper bound since there appears to be an incompatibility
 # between pylinac and scikit-learn version 0.24.0
-# Not needed as PyMedPhys dependency. Included only as a fix.
-scikit-learn = { version = "<0.24.0", optional = true }  # groups = ["user", "dev"]
+scikit-learn = { version = "<0.24.0", optional = true }  # groups = ["user", "dev", "mosaiq"]
```

Comment:
is this all that needs to be changed, so that scikit-learn is available during the MosaiqDbTest?

Reply:
A propagate will be needed, but @pymedphys-bot should be able to fix that for you in a minute or so.

Reply:
Also, is there a reason for the upper pinning?

Reply:
Beautiful 🙂, the tests pass 🙂

Reply:
would you like me to try removing the upper pinning and see if that causes any apparent problems?  are there tests involving pylinac that you think would flag if there were problems?

Reply:
Yup, it should flag the issue in the tests.

Reply:
ok I can try in a separate PR